source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.cpp

```diff
@@ -169,6 +169,28 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
                     // Otherwise, leave all options unset
                     radioButton.IsChecked(IsChoiceSelected(values, input));
                 }
+
+                radioButton.PreviewKeyDown(
+                    [this, stackPanel](winrt::IInspectable const& sender, winrt::KeyRoutedEventArgs const& args) -> void
```

Comment:
is there any reason to pass `this` in the lambda capture?:)

Reply:
Nope, just removed it :)